purchases/src/main/kotlin/com/revenuecat/purchases/google/usecase/GetBillingConfigUseCase.kt

```diff
@@ -42,6 +44,7 @@ internal class GetBillingConfigUseCase(
         if (received == null) {
             onError(PurchasesError(PurchasesErrorCode.StoreProblemError, BillingStrings.BILLING_CONFIG_NULL_ON_SUCCESS))
         } else {
+            deviceCache.setStorefront(received.countryCode)
```

Comment:
This will be updated every time we connect to the billing client. Having said that, there is a chance of the cache to be outdated if it loses connection at some point, but in that case, we won't be able to get offerings or query purchases, so it should be fine on most cases.

Reply:
Yeah yeah, that's totally fine!